scoresheets/Restaurant.tex

```diff
@@ -1,24 +1,16 @@
-The maximum time for this test is 15 minutes.
+The maximum time for this test is 10 minutes.
 
-\small\begin{scorelist}
+\small\begin{scorelist}[startbutton=false]
 	\scoreheading{Main Goal}
-	\scoreitem{500}{Take and serve an order}
-	\penaltyitem[2]{200}{Being guided to the \emph{Kitchen-bar} or to a customer's table}
-	\penaltyitem{100}{Not making eye-contact when taking an order}
-	\penaltyitem[4]{50}{Bypassing object manipulation (handover)}
-	\penaltyitem[5]{100}{Bypassing object manipulation}
+	\scoreitem[2]{500}{Take and serve an order}
+	\penaltyitem{25}{Request help by receiving directions}
```

Comment:
These should all be 2x-ed

Reply:
Some yes. Guiding can happen more times. The robot could request guidance to the table where someone wants to order and back to the bar.